fuse/data/utils/samplers.py

```diff
@@ -36,16 +34,19 @@ def __init__(
         dataset: DatasetBase,
         balanced_class_name: str,
         num_balanced_classes: int,
+        sampler: Optional[Sampler] = None,
```

Comment:
Should it be always None?
I thought that after their fix, we don't need it anymore.